qdrant_client/qdrant_client.py

```diff
@@ -69,9 +69,17 @@ def __init__(self,
             self._init_grpc_collections_client()
 
     def __del__(self):
+        self.close()
+
+    def close(self):
         if self._grpc_channel is not None:
             self._grpc_channel.close()
-        self.openapi_client.client._client.close()
+
+        try:
+            self.openapi_client.client._client.close()
+        except AttributeError as e:
+            if str(e) != "'NoneType' object has no attribute 'CLOSED'":
```

Comment:
That is usually a bad practice to check directly for an error message.
It would be best to have a way to reliably reproduce the error. After which we could wind the cause of this error instead of just patching the consequences.